flow/connectors/postgres/cdc.go

```diff
@@ -386,7 +386,8 @@ func PullCdcRecords[Items model.Items](
 
 	for {
 		if pkmRequiresResponse {
-			if cdcRecordsStorage.IsEmpty() && int64(clientXLogPos) > req.ConsumedOffset.Load() {
+			if cdcRecordsStorage.IsEmpty() && int64(clientXLogPos) > req.ConsumedOffset.Load() &&
+				time.Since(standByLastLogged) >= 1*time.Minute {
```

Comment:
ideally we'd throttle this based on this update, not alongside existing logging code. Or move logging code in here & only log once a minute

As is we may be logging every 10s & never update offset